docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,601 @@
+# CA Deployment steps
+
+## Download the binaries
+
+The Fabric CA server and CA client binaries are available for download from [github](https://github.com/hyperledger/fabric-ca/releases). Scroll down to **Assets** and select the latest binary for your machine. The .zip file contains both the CA server and the CA client binaries. Copy the CA server file to a directory where you will run the server. Likewise, copy the CA client binary to its own directory. Having the CA client in its own folder facilitates certificate management especially when you need to interact with multiple CAs.  When you issue a command from the CA client against a CA server, you can target a specific CA by modifying the CA server URL on the request. Therefore, only a single Fabric CA client binary is required and can be used to transact with multiple CAs. More on using the Fabric CA client below.
+
+After you have mastered the use of these binaries, it is likely you will want to use the Fabric CA image for example in a Kubernetes or Docker deployment. For now though, the purpose of this topic is to teach you how to properly use the binaries.
+
+## Fabric CA client
+
+While you can use the Fabric SDKs to interact with your CA, it is recommended that you **use the Fabric CA client to register and enroll node admin identities**. The instructions provided in this topic assume a single Fabric CA client is being used.
+
+**Tip:** While you can run the Fabric CA client binary from anywhere you want, for ease of following these instructions we will refer to it in its own directory named `fabric-ca-client`.
+
+Because you will use the CA client to submit transactions to the CA for enrolling identities, certificate management is critically important when using the CA client. The private and public keys are first generated locally by the Fabric CA client, and then the public key is sent to the CA which returns an encoded "signed certificate". A best practice therefore is for each TLS CA, organization CA, or intermediate CA that you create, to create the following folders:
+- You need a sub-folder in order to connect to each CA, such as `/tls-ca` or `/org1-ca` or `/int-ca`. This folder can be under the Fabric CA client or anywhere that the CA client can access the path. For purposes of these instructions, these folders reside inside the `fabric-ca-client` directory. For example:
```

Comment:
Since this is the first time you mention intermediate CA in this topic, explain what it will be used for. Is it an intermediate CA whose parent (root) is the org1-ca?  And therefore this topic will interact with the intermediate CA rather than the root org1-ca when enrolling identities?

Reply:
I'm glad you pointed this out. I added this to the introduction where I discuss the binaries:

In this topic, we will use the binaries to deploy three different types of CAs: the TLS CA, an organization CA, and an intermediate CA. The organization CA serves as the root CA or parent server for the intermediate CA. If you have not already, you should review the topic on [Planning for a CA](ca-deploy-topology.html) to understand the purpose of each type of CA and their differences.

We don't actually enroll node identities in this topic (that is coming next in the Using CA topic)  but to your point I added this line to the next steps section at the end of the topic:

If you did configure an intermediate CA, you can now use that CA to register and enroll identities for the organization instead of the root CA.